soc/arm/microchip_mec/mec172x/Kconfig.defconfig.mec172xnsz

```diff
@@ -47,4 +47,8 @@ config PECI_XEC
 	default y
 	depends on PECI
 
+config SPI_XEC_QMSPI_V2
+	default n
```

Comment:
`default n` is the default value for booleans. No need to write it out.

Reply:
using choice now